includes/definitions/discovery/powerwalker.yaml

```diff
@@ -97,7 +95,6 @@ modules:
                     descr: Battery charge remaining
         temperature:
             options:
-                skip_values_lt: 0
```

Comment:
Why remove these?

Reply:
because these sensors can be negative... before these were NOT applying at all (wrong key name), to keep existing behavior, they are removed.

Others that seem sensible to NOT be negative were updated, also checked test data to confirm as much as I can.